src/app/backend/resource/pod/events.go

```diff
@@ -7,11 +7,10 @@ import (
 	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/event"
 	client "k8s.io/client-go/kubernetes"
-	api "k8s.io/client-go/pkg/api/v1"
 )
 
 // Get all events that are associated with this pod.
-func GetEventsForPods(client client.Interface, dsQuery *dataselect.DataSelectQuery, namespace, podName string) (
+func GetEventsForPod(client client.Interface, dsQuery *dataselect.DataSelectQuery, namespace, podName string) (
```

Comment:
Comments for backend methods should start with methods names to fulfill style requirements: `GetEventsForPod gets...`. We should comment all public methods.